crates/control/src/sensor_data_receiver.rs

```diff
@@ -58,12 +59,14 @@ impl SensorDataReceiver {
         context.maximum_temperature.fill_if_subscribed(|| {
             sensor_data
                 .temperature_sensors
-                .as_vec()
                 .into_iter()
-                .flatten()
                 .fold(0.0, f32::max)
         });
 
+        context
+            .total_current
+            .fill_if_subscribed(|| sensor_data.currents.into_iter().sum());
```

Comment:
Are currents always positive?

Reply:
I hope so. But let's test that with hardware 😅
What happens, if the stiffness is set to zero and the motor is moved by an external force?

Reply:
Yes, currents are always positive